WordPress/Classes/Login/LoginWithUrlView.swift

```diff
@@ -16,25 +16,38 @@ struct LoginWithUrlView: View {
     @State private var urlField: String = ""
     @State private var isLoading = false
 
+    private var isContinueButtonDisabled: Bool {
+        isLoading || urlField.trim().isEmpty
+    }
+
     init(
+        client: LoginClient,
         anchor: ASPresentationAnchor,
         loginCompleted: @escaping (WordPressOrgCredentials) -> Void
     ) {
-        self.client = LoginClient(session: URLSession(configuration: .ephemeral))
+        self.client = client
         self.anchor = anchor
         self.loginCompleted = loginCompleted
     }
 
     var body: some View {
         VStack(alignment: .leading) {
-            Text("Enter the address of the WordPress site you'd like to connect.").padding(.vertical)
+            VStack(alignment: .center, spacing: 16) {
+                Image("splashLogo")
+                    .resizable()
+                    .aspectRatio(contentMode: .fit)
+                    .frame(width: 60, height: 60)
+                    .padding(24)
+                Text(Self.enterSiteAddress)
+            }
 
             TextField(text: $urlField) {
                 Text("example.com")
             }
             .padding(.vertical)
             .overlay(Divider(), alignment: .bottom)
```

Comment:
The underline is too far from the view and looks like a separate divider. It was the case in the previous implementation also.